Model/AbstractArticle.php

```diff
@@ -181,7 +180,7 @@ public function getAbstract()
     /**
      * {@inheritdoc}
      */
-    public function setCategories(Collection $categories = null)
+    public function setCategories(ArrayCollection $categories = null)
```

Comment:
I didn't notice that one. I don't think you should change to `ArrayCollection`, b/c it's better to type hint against an interface.